view_conversations/conv-viewer-webapp/src/app/app.component.ts

```diff
@@ -83,9 +83,9 @@ export class AppComponent implements OnInit {
     let searchBy = CONVERSATION_ID_TEXT;
     let searchFor = '';
 
-    let browseBy = MOST_TOXIC_TEXT;
-    let browseUpper = 1;
-    let browseLower = 0;
+    const browseBy = MOST_TOXIC_TEXT;
+    const browseUpper = 1;
+    const browseLower = 0;
```

Comment:
I actually changed this part in an upcoming commit, this is what I did wrong with locationHash.